src/conversion/modelconversiondispatcher.js

```diff
@@ -389,6 +396,14 @@ export default class ModelConversionDispatcher {
 		return consumable;
 	}
 
+	_createRangeConsumable( range ) {
```

Comment:
I'll change to `_createMarkerConsumable` because it will be only for one marker at a time.